app/main.py

```diff
@@ -1,8 +1,39 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class: int, clean_mark: int,
+                 brand: str) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self, distance_from_city_center: float, clean_power: int,
+                 average_rating: float, count_of_ratings: int) -> None:
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def serve_cars(self, car_collection:
+                   list) -> float:
+        benefit = []
+        for car in car_collection:
+            if car.clean_mark < self.clean_power:
+                benefit.append(self.calculate_washing_price(car))
+                self.wash_single_car(car)
+        return round(sum(benefit), 1)
```

Comment:
```suggestion
        benefit = 0
        for car in car_collection:
            if car.clean_mark < self.clean_power:
                benefit += self.calculate_washing_price(car)
                self.wash_single_car(car)
        return round(benefit, 1)
```
no need to use loop in this case